updates/testing.json

```diff
@@ -0,0 +1,19 @@
+{
+  "updates": {
+    "barriers": [],
+    "deadends": [
+      {
+        "version": "30.20190716.1",
+        "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/215"
+      }
+    ],
+    "rollouts": [
+      {
+        "version": "30.20190801.0",
+        "start_epoch": "1565017200",
+        "start_value": "0.0",
+        "duration_minutes": "120"
```

Comment:
Why isn't this field and `start_epoch` actual integers instead? We might've discussed this but I forgot. :)
E.g. makes it easier to parse, schema check, etc...

Reply:
A copy of these is forwarded to the graph, and I can only have strings there. Here we are free to either have strings everywhere too, or proper types to parse and forward to the graph.
I have no preferences and just went for the most similar format.

For reference, the resulting graph metadata looks like this:
```
{
  "version": "30.20190801.0",
  "metadata": {
    "<prefix>.start_epoch": "1565017200"
  },
}
```

Reply:
Hmm, I have a preference for using the proper types here if we can rather than being forced to use strings for everything that will also appear in the graph. @bgilbert Thoughts?

Reply:
In general I'd prefer proper types, but if that complicates the translation too much, I can live with strings.  I'm happy to leave the decision to @lucab.  :slightly_smiling_face:

Reply:
Covered this out-of-band with @jlebon: we'll go with proper types.